lib/hawkular/hawkular_client.rb

```diff
@@ -47,13 +31,37 @@ def method_missing(name, *args, &block)
       delegate_client.__send__(method, *args, &block)
     end
 
+    def inventory
+      @inventory ||= Inventory::InventoryClient.create(entrypoint: "#{@state[:entrypoint]}/hawkular/inventory",
```

Comment:
Why does `Metrics` has a different class name? Can we change the others to be like it? I like it better than to have `Inventory::Inventory` and `Alerts::Alerts`.